src/molecular_orbital_hamiltonian.jl

```diff
@@ -32,19 +32,19 @@ function molecular_orbital_hamiltonian_coefficients(;
   return (; hα, gα, nαocc, hartree_fock_energy, nuclear_energy)
 end
 
-function _molecular_orbital_hamiltonian(hα, gα, nuclear_energy)
+function _molecular_orbital_hamiltonian(hα, gα, nuclear_energy; cutoff = 1e-15)
```

Comment:
@mtfishman do you think this may be confusing cause of the meaning of `cutoff` in contraction functions?

Reply:
In Julia functions like `isapprox`, the standard name for this is `rtol` and `atol`:
```
  isapprox(x, y; atol::Real=0, rtol::Real=atol>0 ? 0 : √eps, nans::Bool=false[, norm::Function])


  Inexact equality comparison. Two numbers compare equal if their relative distance or their absolute
  distance is within tolerance bounds: isapprox returns true if norm(x-y) <= max(atol, rtol*max(norm(x),
  norm(y))). The default atol is zero and the default rtol depends on the types of x and y. The keyword
  argument nans determines whether or not NaN values are considered equal (defaults to false).
```

We could call it `atol`, anticipating that we could add `rtol` which would be defined as something like `norm(hα[i, j]) / maximum(norm, hα) > atol`.

Reply:
yeah I think this is a good option